x-pack/plugin/security/qa/multi-cluster/build.gradle

```diff
@@ -5,129 +5,11 @@
-  if (BuildParams.random.nextBoolean()) {
-    setting 'xpack.security.remote_cluster_server.ssl.enabled', 'true'
-  }
-  setting 'xpack.security.remote_cluster_server.ssl.key', 'remote-cluster.key'
-  setting 'xpack.security.remote_cluster_server.ssl.certificate', 'remote-cluster.cert'
-  keystore 'xpack.security.remote_cluster_server.ssl.secure_key_passphrase', 'remote-cluster-password'
-  // client auth defaults to none for remote_cluster_server
-  if (BuildParams.random.nextBoolean()) {
-    setting 'xpack.security.remote_cluster_server.ssl.client_authentication', 'none'
-  }
-
-  // The remote cluster client SSL can be enabled or disable independently from the server
-  if (BuildParams.random.nextBoolean()) {
-    setting 'xpack.security.remote_cluster_client.ssl.enabled', 'false'
-  }
-
-  user username: "test_user", password: "x-pack-test-password"
-}
-
-def queryingCluster = testClusters.register('querying-cluster') {
-  requiresFeature 'es.untrusted_remote_cluster_feature_flag_registered', Version.fromString("8.5.0")
-
-  module ':modules:analysis-common'
-  module ':modules:lang-painless'
-  module ':modules:data-streams'
-  module ':x-pack:plugin:mapper-constant-keyword'
-  module ':x-pack:plugin:async-search'
-
-  setting 'xpack.security.enabled', 'true'
-  setting 'xpack.license.self_generated.type', 'trial'
-  setting 'cluster.remote.connections_per_cluster', "1"
-
-  extraConfigFile 'transport.key', file("src/test/resources/ssl/transport.key")
-  extraConfigFile 'transport.cert', file("src/test/resources/ssl/transport.crt")
-  extraConfigFile 'transport.ca', file("src/test/resources/ssl/transport-ca.crt")
-
-  setting 'xpack.security.transport.ssl.enabled', 'true'
-  setting 'xpack.security.transport.ssl.key', 'transport.key'
-  setting 'xpack.security.transport.ssl.certificate', 'transport.cert'
-  setting 'xpack.security.transport.ssl.key_passphrase', 'transport-password'
-  setting 'xpack.security.transport.ssl.certificate_authorities', 'transport.ca'
-  setting 'xpack.security.transport.ssl.client_authentication', 'required'
-  setting 'xpack.security.transport.ssl.verification_mode', 'certificate'
-
-  // Client (QC) side ssl configuration for remote cluster
-  // Remote cluster client SSL is enabled by default
-  if (BuildParams.random.nextBoolean()) {
-    setting 'xpack.security.remote_cluster_client.ssl.enabled', 'true'
-  }
-  extraConfigFile 'remote-cluster.ca', file("src/test/resources/ssl/remote-cluster-ca.crt")
-  setting 'xpack.security.remote_cluster_client.ssl.certificate_authorities', 'remote-cluster.ca'
-
-  setting 'cluster.remote.my_remote_cluster.mode', 'proxy'
-  setting 'cluster.remote.my_remote_cluster.proxy_address', {
-    "\"${fulfillingCluster.get().getAllRemoteAccessPortURI().get(0)}\""
-  }
-  setting 'cluster.remote.my_remote_cluster.authorization', 'foobar'
-
-  user username: "test_user", password: "x-pack-test-password"
+dependencies {
+  javaRestTestImplementation project(':x-pack:plugin:core')
```

Comment:
Why do we need the dependencies on these two x-pack modules? I don't see any references to them.

Reply:
Err they were copy/paste from ServiceAccountIT. Removed.